marlowe-cli/test/non-pab/inline/trivial.yaml

```diff
@@ -1,4 +1,4 @@
-stTestName: Sample Test
+stTestName: Inline Simple Contract
```

Comment:
Will we be able to retire the use of `marlowe-cli/run-nonpab-tests.sh` in favor of the corresponding equivalent tests in this DSL? Could we have `run-nonpab-tests.sh` execute the DSL versions of the tests instead of the `bash` versions?

Reply:
> Will we be able to retire the use of marlowe-cli/run-nonpab-tests.sh in favor of the corresponding equivalent tests in this DSL?

Yes we have ongoing upgrade of the test suite to cover all the cases: https://input-output.atlassian.net/browse/SCP-4456

> Could we have run-nonpab-tests.sh execute the DSL versions of the tests instead of the bash versions?

For sure we want to create a simple script which runs DSL based test suite: https://input-output.atlassian.net/browse/SCP-4460

Brian, do we want to really drop this "literate" test cases (they test really the validity of the examples code)?